Phaethon/Tests/IntegrationTests/ItemTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Core.Model;
+using Newtonsoft.Json;
+
+namespace Tests.IntegrationTests
+{
+    public class ItemTest: InternalTestFakeServerBase
+    {
+        #region GetItem
+        [Test]
+        public async Task GetItem_CorrectItemId_IsSuccessStatusCodeAndItemReturned()
+        {
+            //Setup
+            Element element = InvoiceTest.GetElementSeed();
+            var parameters = HttpUtility.ParseQueryString(string.Empty);
+            parameters["id"] = element.Item.ID.ToString();
+
+            //Act
+            var response = await _client.GetAsync("Item/GetItem?" + parameters);
+            Item item = JsonConvert.DeserializeObject<Item>(await response.Content.ReadAsStringAsync());
+
+            //Assert
+            Assert.IsTrue(response.IsSuccessStatusCode);
+            Assert.AreEqual(true, element.Item.ID == item.ID &&
```

Comment:
You have so much duplicate code here. Take a look at my implementation in Job.cs where I implement the .Equals(); method, to tell C# how to compare 2 objects.

Reply:
i thought .equals didnt work for you

Reply:
I see you didnt actually review my previous PR's you just approved them blindly XD

but yeah, it works.